Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -224,14 +227,22 @@ class HomeFragmentViewModel @AssistedInject constructor(
     ) { tracingItem, submissionItem, submissionState, statsData ->
         mutableListOf<HomeItem>().apply {
             when (submissionState) {
-                TestPositive, SubmissionDone -> {
+                TestPositive, is SubmissionDone -> {
```

Comment:
What does this change do?

Reply:
`SubmissionDone` is now data class not an object, so we need to change this syntax.

Reply:
I've changed SubmissionDone from an object to a data class so we can no longer use it without explicitly putting the is operator before it.